docs/General/Meetings/Radiomics_meeting.md

```diff
@@ -0,0 +1,16 @@
+# Radiomics Meeting
+## Time and Room
+Radiomics meeting happens every 2 weeks on Tuesday, 2:00 - 3:00pm at PMCRT 11-710
+
+## Agenda
+Flash updates - 5-10 minutes/person on what they’re working on  
+Q&A for project related challenges  
+Award/presentation updates, if any  
+Data update: New ORCESTRA announcements for datasets, dataset suggestions etc.  
+Grant updates
+
+## Attendees
+The attendance is mandatory for Radiomics people
+
+## Resources
+The meeting notes need to be filled out for every meeting at [Radiomics meeting - coverage](https://docs.google.com/document/d/1zlbdjlhEnUrWQExUBrYL56z_FiCE_Dd6nHSAHpGnk5o/edit?usp=sharing)
```

Comment:
A minor wording change might help here, as well as a period at the end of the sentence.

```suggestion
The meeting notes need to be filled out for every meeting in the [Radiomics meeting - coverage](https://docs.google.com/document/d/1zlbdjlhEnUrWQExUBrYL56z_FiCE_Dd6nHSAHpGnk5o/edit?usp=sharing) document.
```